app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```

Comment:
- [ ] for this controller the index and the post actions should have been created.